content/publication_resources/bibtex/lahti.bib

```diff
@@ -116,6 +165,33 @@ @Article{Khalighi2023
 }
 
 
+@Article{Vaura2022,
+  author =	 {Felix Vaura, Hyunkyung Kim, Miriam S Udler; FinnGen; 
+                  Veikko Salomaa, Leo Lahti, Teemu Niiranen },
+  title =	 {Multi-Trait Genetic Analysis Reveals Clinically Interpretable
+                    Hypertension Subtypes},
+  journal =	 {Circulation: Genomic and Precision Medicine.},
+  year =	 2022,
+  doi =		 {10.1161/CIRCGEN.121.003583},
```

Comment:
always add volume, number/issue, and page information if available.

for this, journal page seems to show at least volume and number

sometimes you can find citation info from the manuscript page as well and those details are there as well.

add these informations for this and the other new references